ts/screens/wallet/ConfirmToProceedTransactionScreen.tsx

```diff
@@ -0,0 +1,153 @@
+/**
+ * This screen ask the authorization to proceed with the transaction.
+ * TODO: integrate credit card componet for visualization of the payment method selected for the transaction
+ */
+
+import {
+  Body,
+  Button,
+  Container,
+  Content,
+  H1,
+  Icon,
+  Left,
+  Text,
+  View
+} from "native-base";
+import * as React from "react";
+import { StyleSheet } from "react-native";
+import { Col, Grid, Row } from "react-native-easy-grid";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import { WalletStyles } from "../../components/styles/wallet";
+import AppHeader from "../../components/ui/AppHeader";
+import PaymentBannerComponent from "../../components/wallet/PaymentBannerComponent";
+import I18n from "../../i18n";
+type Props = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+}>;
+
+const styles = StyleSheet.create({
+  space: {
+    justifyContent: "space-between"
+  }
+});
+
+export class ConfirmToProceedTransactionScreen extends React.Component<
+  Props,
+  never
+> {
+  constructor(props: Props) {
+    super(props);
+  }
+
+  private goBack() {
+    this.props.navigation.goBack();
+  }
+
+  public render(): React.ReactNode {
+    const IMPORTO = "€ 199,00";
+    const FEE = "€ 0,50";
+    const TOTALEALL = "€ 199,50";
+
+    return (
+      <Container>
+        <AppHeader>
+          <Left>
+            <Button transparent={true} onPress={() => this.goBack()}>
+              <Icon name="chevron-left" />
+            </Button>
+          </Left>
+          <Body>
+            <Text>{I18n.t("wallet.ConfirmPayment.header")}</Text>
+          </Body>
+        </AppHeader>
+
+        <Content original={true}>
+          <PaymentBannerComponent navigation={this.props.navigation} />
+          <View style={WalletStyles.paddedLR}>
+            <View spacer={true} large={true} />
+            <H1>{I18n.t("wallet.ConfirmPayment.askConfirm")}</H1>
+            <View spacer={true} large={true} />
+            <Text>CREDIT CARD COMPONENT</Text>
+            <View spacer={true} large={true} />
+            <Grid>
+              <Row>
+                <Col>
+                  <Text>{I18n.t("wallet.ConfirmPayment.partialAmount")}</Text>
+                </Col>
+                <Col>
+                  <Text bold={true} style={WalletStyles.textRight}>
+                    {IMPORTO}
+                  </Text>
+                </Col>
+              </Row>
+              <Row>
+                <Col size={4}>
+                  <Text>
+                    {I18n.t("wallet.ConfirmPayment.fee")}
+                    <Text>{""} </Text>
```

Comment:
Already mentioned somewhere else by @cloudify i believe.
I would personally recommend using [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for the purpose:
```
<Text>{`${I18n.t("wallet.ConfirmPayment.fee")} `}</Text>
```